RoyaltiesProcessorDelivery/readme.md

```diff
@@ -99,16 +100,9 @@ Until properly handled, these assumptions may result in incorrect payment estima
   - Current behavior of subgraph is that only the last sale to occur in a tx is recorded (overwrites all previous sales in tx)
 
 **Overestimates Artist Payments**:
-- (when NOT using OpenSea API mode*) Assumes bundled sales ONLY contain Art Blocks flagship or PBAB pieces
-  - Royalties collected in bulk-sales are divided by total number of tokens in the sale. Our subgraph only tracks Art Blocks tokens. When a bulk sale contains both Art Blocks token(s) *and* non-Art Blocks tokens, the script divides total royalty by the qty of Art Blocks tokens instead qty of all tokens.
-  - Typically ~small portion of total sales
-  - Solution to this is to either change the behavior of public subgraph's handling of bulk sales, or use a different source for bulk transactions that contain Art Blocks tokens.
-    - Different sources could be: parse logs via web3 calls for *every* transaction (significant usage of web3 provider), or trust OpenSea's api and get sale information for every transaction. The subgraph may be able to indicate bulk transacitons, which would reduce queries.
-- (when NOT using OpenSea API mode*) Assumes royalties are being collected in bulk private sales at or after block `13147635`
-  - In reality, it appears that OpenSea only collects artist royalties on bundle sales that also contain tokens from a single OpenSea collection
-  - TBD if OpenSea decides to change royalty policies in the future
-
-*-When using OpenSea API mode, results appear to be entirely accurate because OpenSea excludes sale events from our queries when artist royalties were not collected.
+  - *no known bugs at this time*
+
+*-When using OpenSea API mode, there are no known bugs at this time*

```

Comment:
May be worth expanding on this with a "if you discover a new previously unknown bug when using OpenSea API mode, please file a bug" with a link to the area to file a bug in this repo, given that it is public.

Reply:
🙏